src/functinalities.js

```diff
@@ -0,0 +1,56 @@
+let mylist = JSON.parse(localStorage.getItem('mylist')) || [];// eslint-disable-line
+
+const listGroup = document.querySelector('.todo-group');
+const newTask = document.querySelector('.todo-add').querySelector('input');
+const getmylist = () => {
+  const mylistElement = mylist.map((item) => `
+        <li class="todo-list todo-item" id=${item.index}>
+          ${item.completed === true ? `
+            <i class="fa-solid fa-check checked-icon"></i>`
+    : '<i class="fa-solid fa-square unchecked-icon"></i>'}
+          <input type="text" class=${item.completed === true ? 'decoration edit-todo' : ' edit-todo'}  value="${item.description}">
+          <span class="edit-focus-element"></span>
+          <i class="fa-solid fa-trash-can delete-icon"></i>
+          <i class="fa-solid fa-ellipsis-vertical more-icon"></i>
+        </li>`).join('');
+  listGroup.innerHTML = mylistElement;
+  return listGroup;
+};
+
+const addmylist = (event) => {
+  if (newTask.value === '') return;
+  if (event.key === 'Enter' || event === 'clicked') {
+    const todoElement = {
+      description: newTask.value,
+      completed: false,
+      index: mylist.length + 1,
+    };
+    mylist = [...mylist, todoElement];
+    localStorage.setItem('mylist', JSON.stringify(mylist));
+    getmylist();
+  }
+};
+
+const editmylist = ({ index, event }) => {
+  if (event.target.value === '') return;
+  if (event.key === 'Enter') {
+    mylist[index].description = event.target.value;
+    localStorage.setItem('mylist', JSON.stringify(mylist));
+  }
+};
```

Comment:
- [ ] Kindly try to fix the edit functionality when I edit the last value the local storage doesn't update.
![image](https://user-images.githubusercontent.com/24703171/171456407-a30c1d2b-88df-4e3e-aceb-87b69c63101a.png)
and when I edit a value before it, it's updated in the wrong way.
![image](https://user-images.githubusercontent.com/24703171/171458451-5b6be05e-7f1d-4780-8d75-95e0bd30531f.png)
it's updated the second value instead of the first value.